java-package/integration/build.gradle

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id 'application'
+    id 'java'
+}
+
+group 'incubator-mxnet.java-package'
+version '0.0.1-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+application {
+    mainClassName = System.getProperty("main", "org.apache.mxnet.integration.IntegrationTest")
+}
+
+dependencies {
+    api "commons-cli:commons-cli:${commons_cli_version}"
+    api "org.apache.logging.log4j:log4j-slf4j-impl:${log4j_slf4j_version}"
+
+    api project(":mxnet-engine")
+    implementation 'org.testng:testng:7.1.0'
+//    testImplementation(":mxnet-engine")
+    testImplementation("org.testng:testng:${testng_version}") {
+        exclude group: "junit", module: "junit"
+    }
+    testImplementation "org.slf4j:slf4j-simple:${slf4j_version}"
+}
+
+run {
+    environment("TF_CPP_MIN_LOG_LEVEL", "1") // turn off TensorFlow print out
```

Comment:
For TF use only

Reply:
> can you do
> 
> ```
> git submodule update --init --recursive
> ```
> 
> ?

- Solved this problem by hard reset the version of these submodules, after some attempts 😄 .
- Removed this TF used only environment variable.